### ISSUE TITLE
Entities problems / Can't spawn IG

### ISSUE THREAD

**Issue by L4BORG** [tastybento/askyblockIssues#865]
Thank you for filing a bug report. Please complete these sections to help speed resolution.

**Server version? Remember to say if it is Spigot/Bukkit/Paper etc.**
PaperSpigot 1.8.8.

**Plugin version:**
Latest

**Is this a new install or upgrade from earlier plugin version? If upgrade, what version did you use before?**

**(Optional) What other plugins are you using now? (do /plugins and paste text from the console log here)**

**Description of issue. What happened?**
Can't spawn iron golems with ironblocks and pumpkins. Also it seems like sometimes we would get ghost entities on ASkyBlock server - entities are are not there but can't not be removed.

We're not using builtin entity limited (commented out of config)

**Steps to make this happen**

1. 
2.
3.

**What do you think should happen?**

**Comment by FlailoftheLord** [comment]
Yeah ive experienced the same bug on several servers, it seems to be a conflict between 1.8 servers and 1.12 clients... also doesnt make Snow Golems

**Comment by tastybento** [comment]
I'm not sure there's much I can do about this. Is there anything printed in the console log when you try to make a golem?

**Comment by L4BORG** [comment]
No, but we are able to spawn snow golems on latest PaperSpigot 1.8 but not IronGolems. Also the main issue are ghosted entities.

Would it be possible to add config option that would entirely disable entity caps?

**Comment by tastybento** [comment]
> Would it be possible to add config option that would entirely disable entity caps?

I'm not sure what you mean by entity caps, but if you don't want any entity limits, then just don't put anything in the entitylimits: section in config.yml.

**Comment by L4BORG** [comment]
That's what we're using but it seems something is still messing with entities.

**Comment by FlailoftheLord** [comment]
Are you using any other mob modifiers like EssentialsX or WorldGuard or Mythic mobs.... the list goes on, any plugin that messes with mob spawning or has options to disable or enable certain mobs from spawning.. ect...

**Comment by L4BORG** [comment]
Nothing, except for custom plugin that would cap. max no. of mobs per chunk - and even without this plugin, the problem persists.
